3-platform/delivery/.gitignore

```diff
@@ -0,0 +1,38 @@
+target/
```

Comment:
there is already a present gitignore in the root folder (./6core/.gitignore). 
Modify it for reusing purposes.

Reply:
@sokimaaa , does it mean that we'll have only one .gitignore file for all microservices?